edelphi/src/main/java/fi/metatavu/edelphi/FeatureNotAvailableOnSubscriptionLevelException.java

```diff
@@ -0,0 +1,31 @@
+package fi.metatavu.edelphi;
+
+import java.util.Locale;
+
+import fi.metatavu.edelphi.domainmodel.users.SubscriptionLevel;
+import fi.metatavu.edelphi.i18n.Messages;
+import fi.metatavu.edelphi.smvcj.SmvcRuntimeException;
+
+/**
+ * Exception that is thrown when requested feature is not 
+ * available on current user's subscription level
+ */
+public class FeatureNotAvailableOnSubscriptionLevelException extends SmvcRuntimeException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Rename this class to remove "Exception" or correct its inheritance. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2166)